packages/atomic/src/global/global.pcss

```diff
@@ -7,6 +7,10 @@
   @apply font-sans font-normal text-base;
 }
 
+:host(.atomic-without-values) {
+  display: none;
+}
```

Comment:
I would prefer if this was moved to a file that is specifically for facets.

Reply:
I don't know if I should be more generic here, I use that class both for facet & breadcrumb, basically hides the element's content. What about renaming to atomic-hidden and atomic-visible (for atomic-with-values) ?  This would be another v1 todos

Reply:
(added a comment)

Reply:
alright, sounds good